cEP-0020.md

```diff
@@ -0,0 +1,465 @@
+
+Implementing the above privileges is beyond the scope of the gamification system.
+The gamification system will merely expose a list of privileges that each
+person has, and other systems i.e.: [corobo](https://github.com/coala/corobo)
+are expected to adapt to use these privileges.
+
+While implementing this process, a first manual check should be done for testing
+purpose, and then automated only if there is no serious risk or there is a good
+risk reaction mechanism in place.
+
+### Badges
+
+Another way of explicitly nudging a newcomer to action is to award badges for
+completing tasks. Badges enable newcomers to follow their performance and compare
+to other newcomers.
+Badges will be awarded after performing certain activities by the newcomers:
+
+- Badge-I:
+  - Name: The bug finder
+  - Details: The one who find bugs in the existing codebase
+  - Activities:
+    - Create at least one issue which labeled as bug
+- Badge-II:
+  - Name: The bear hunter
+  - Details: The one who create issues about new bears
+  - Activities:
+    - Create at least one bear proposal issue
+- Badge-III:
+  - Name: The bear writer
+  - Details: The one who write new bears
+  - Activities:
+    - Merged at least 2 bear proposal PRs
+- Badge-IV:
+  - Name: The helper
+  - Details: The one who answers every newcomer questions on Gitter
+  - Activities:
+    - At least 20 good messages to newcomer
+- Badge-V:
+  - Name: The coala Developer
+  - Details: The ones who are in the coala developers team
+  - Activities:
+    - Run coala on a project of yours
+    - Merge a difficulty/newcomer Pull Request
+    - Review at least a difficulty/newcomer Pull Request
+    - Merge a difficulty/low Pull Request
+    - Review at least a difficulty/low or higher Pull Request
+
+So, for becoming a developer at coala, a newcomer must complete all the
+activities mentioned in the `coala Developer` badge. But in case a newcomer
+is not able to become a developer at coala, he/she could take away the other
+badges.
+
+Names and metrics of tracked activities will also be shown on the user
+dashboard along with the number of times they have been done. That will
+help maintainers to check if an automatically awarded badge is correct or
+not. So, in case of doubts on any newcomer, there will be a button to
+revoke those badges.
+
+The `coala Developer` badge doesn't mean that a newcomer is in the coala
+developer team. If a newcomer wants to be the part of the developer team
+after earning the developer badges, he/she will have to ask a maintainer
+on Gitter to verify all the activities they have done, on the dashboard.
+
+## Implementation
+
+### Getting the data
+
+#### GitHub/GitLab
+
+For getting the data related to newcomers activity on GitHub/GitLab to the
+community, this system will mainly use
+[IGitt](https://gitlab.com/gitmate/open-source/IGitt).
+
+#### Gitter
+
+For getting the stats of newcomer's messages, this system will use
```

Comment:
`newcomers'` assuming you're talking about multiple newcomers here. Otherwise "getting the stats of a newcomer's messages"